samples/cpp/tutorial_code/snippets/core_various.cpp

```diff
@@ -39,13 +39,20 @@ int main()
     //! [Algorithm]
 
     //! [RotatedRect_demo]
-    Mat test_image(200, 200, CV_8UC3, Scalar(0));
+    Mat test_image(400, 400, CV_8UC3, Scalar(0));
     RotatedRect rRect = RotatedRect(Point2f(100,100), Size2f(100,50), 30);
 
     Point2f vertices[4];
     rRect.points(vertices);
-    for (int i = 0; i < 4; i++)
-        line(test_image, vertices[i], vertices[(i+1)%4], Scalar(0,255,0), 2);
+
+    putText(test_image, "1", vertices[1], FONT_HERSHEY_SIMPLEX, 2, Scalar(255,255,255));
```

Comment:
https://github.com/opencv/opencv/pull/23342#discussion_r1136043805
Will this param `fontScale=2` produce a huge number on this `400x400px` image?